### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by blueicehaller**
link to JSON API has been changed

**Summary**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of web configuration, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [ ] No

If yes, please describe the impact and migration path for existing setups:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's body (e.g. `Fixes: #xxx[,#xxx]`, where "xxx" is the issue number)

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated (docs/docs/en)
- [ ] Related tests have been updated

**PLEASE DON'T FORGET TO ADD YOUR CHANGES TO CHANGELOG.MD**
- [ ] Yes, CHANGELOG.md is also updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

**Comment by Paulchen-Panther** [comment]
Could you summarize your PRs (#1773, #1774, #1775, #1776, #1777, #1778) into one?
Thank you very much.

**Comment by blueicehaller** [comment]
I hope I have done it correctly. Here are my changes:

### origin/patch-2
Google Play link is no longer available https://play.google.com/store/apps/details?id=nl.hyperion.hyperionpro .
As an alternative you can delete this text:
like our [Android app](https://play.google.com/store/apps/details?id=nl.hyperion.hyperionpro))
27 `* Priority channels are not coupled to a specific led data provider which means that a provider can post led data and leave without the need to maintain a connection to Hyperion. This is ideal for a remote application (like our former [Android app](https://play.google.com/store/apps/details?id=nl.hyperion.hyperionpro), which is no longer available).`

### origin/patch-3
link to Hardware LED devices has been changed
34 `You can find a list of supported hardware [here](https://docs.hyperion-project.org/user/leddevices/Overview.html).`

### origin/patch-4
link to Documentation Configuration has been changed
54 `- [Configuration](https://docs.hyperion-project.org/user/Configuration.html)`

### origin/patch-5
link to Effect development has been changed
55 `- [Effect development](https://docs.hyperion-project.org/effects/Effects.html)`

### origin/patch-6
link to JSON API has been changed
56 `- [JSON API](https://docs.hyperion-project.org/json/JSON.html)`

### origin/patch-1
Installation link has been changed to Getting started
67 `See [Documentation](https://docs.hyperion-project.org/user/GettingStarted.html) or on the [Release Repository](https://releases.hyperion-project.org).`

**Comment by Paulchen-Panther** [comment]
Then please close all pull requests and open a new request that points to your master branch.

Thank you very much
